test/unit/mongoose-cleaner-test.js

```diff
@@ -48,7 +48,7 @@ describe('test/unit/mongoose-cleaner-test.js', () => {
         document.toObject.should.be.calledOnce();
         document.toObject.firstCall.args[0].should.have.properties({
           getters: true,
-          virtuals: false,
+          virtuals: true,
```

Comment:
Did you verify that this didn't return an `id` field everywhere? Because that's probably going to break a lot of places...